R/lasso-development.R

```diff
@@ -26,6 +24,38 @@
 #' F leads to removal of rows containing NULLs.
 #' @param debug Provides the user extended output to the console, in order
 #' to monitor the calculations throughout. Use T or F.
+#' @param cores Number of cores you'd like to use. Defaults to 2.
```

Comment:
Same comment as in LassoDeployment - Docs describe cores as set to 2 by default but examples specify 1 core. Consistency might be good.